opennms-base-assembly/src/main/filtered/etc/report-templates/subreports/ResponseTimeSummary_subreport.jrxml

```diff
@@ -99,19 +99,19 @@
 					<property name="local_mesure_unity" value="pixel"/>
 					<property name="com.jaspersoft.studio.unit.y" value="px"/>
 				</reportElement>
-				<textFieldExpression><![CDATA[String.format("%.2f", $V{ICMP_Latency_MIN})]]></textFieldExpression>
+				<textFieldExpression><![CDATA[String.format("%.2f", $V{ICMP_Latency_MIN}/new Double("1000.0"))]]></textFieldExpression>
```

Comment:
Nit: This would be a little more concise and save having to parse the double from a string:
```suggestion
				<textFieldExpression><![CDATA[String.format("%.2f", $V{ICMP_Latency_MIN}/1000.0d)]]></textFieldExpression>
```

Reply:
`new Double("1000.0")` mimics the way it is currently done in the other database reports.  I can change it here, but it would leave this report inconsistent with the rest.

Reply:
We can keep it as is then - this isn't a code path that is executed very often anyhow.